scipy/ndimage/tests/test_splines.py

```diff
@@ -1,11 +1,18 @@
 """Tests for spline filtering."""
-import numpy as np
 import pytest
 
-from numpy.testing import assert_almost_equal
+import numpy as np
+from scipy._lib._array_api import assert_almost_equal
 
 from scipy import ndimage
 
+from scipy.conftest import array_api_compatible
+skip_xp_backends = pytest.mark.skip_xp_backends
+pytestmark = [array_api_compatible, pytest.mark.usefixtures("skip_xp_backends"),
+              # XXX: only CuPy delegation is implemented
+              # skip_xp_backends("jax.numpy", "torch"),
```

Comment:
what is the story with JAX and PyTorch now?

Reply:
Yup, removed.